### ISSUE TITLE
typhonjs-escomplex v0.1.0 causing errors

### ISSUE THREAD

**Issue by estherleah** [the-simian/es6-plato#95]
Thanks for the upgrade. However, I am getting the following error which is fixed by downgrading typhonjs-escomplex back to version 0.0.12.
```
$ ./node_modules/.bin/es6-plato -r -d ./report src/
lodash.templateSources[0]:15
((__t = ( report.complexity.aggregate.complexity.sloc.physical )) == null ? '' : __t) +
                                                 ^

TypeError: Cannot read property 'sloc' of undefined
    at eval (lodash.templateSources[0]:15:50)
    at writeFileReport (/Users/.../node_modules/es6-plato/lib/plato.js:322:41)
    at /Users/.../node_modules/es6-plato/lib/plato.js:174:6
    at Array.forEach (<anonymous>)
    at runReports (/Users/.../node_modules/es6-plato/lib/plato.js:111:9)
    at /Users/.../node_modules/es6-plato/lib/plato.js:189:4
    at /Users/.../node_modules/es6-plato/node_modules/fs-extra/lib/mkdirs/mkdirs.js:56:16
    at callback (/Users/.../node_modules/graceful-fs/polyfills.js:289:20)
    at FSReqCallback.oncomplete (fs.js:166:5)
error Command failed with exit code 1.
```

**Comment by the-simian** [comment]
Thanks I'll downgrade that and republish today

**Comment by the-simian** [comment]
@estherleah I published 1.2.2 and that solves this! thank you so much for your issue , I'll close this.
